src/components/Board/Output/SymbolOutput/Scroll/Scroll.js

```diff
@@ -23,20 +21,16 @@ export class Scroll extends PureComponent {
       ...other
     } = this.props;
 
-    const scrollStyle = { direction: invertDir(direction) };
```

Comment:
invert the scroll direction is necessary in case of languages written from right to left (Arabic, Hebrew )

Reply:
This is already contemplated on the Scroll__container. These lines were a part of the old way used to auto-scroll to the last element.